### PR TITLE
RC-v1.8 <-- RC-v1.7.1

### PR DESCRIPTION
**Pull request by ap-justin**
Ticket(s):
-

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/pages/Registration/Steps/Documentation/index.tsx [hunk 1]
```diff
@@ -14,7 +15,12 @@ function Documentation() {
   const methods = useForm<FormValues>({
     resolver: yupResolver(schema),
     defaultValues: doc
-      ? { ...(({ level, ...doc }) => doc)(doc) }
+      ? {
```
Comment (misicnenad) [inline src/pages/Registration/Steps/Documentation/index.tsx:18]: Github auto-merged "active countries" & "hq country" commits in a way that breaks forms that use those fields (like this one), created a PR into this one to address this https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1796

**Review by misicnenad — CHANGES_REQUESTED** [review]

File: src/pages/Registration/Steps/Documentation/index.tsx (hunk 1 above)
Comment (ap-justin) [reply src/pages/Registration/Steps/Documentation/index.tsx]: @misicnenad, this error is already fixed in #1790

File: src/pages/Registration/Steps/Documentation/types.ts [hunk 2]
```diff
@@ -1,3 +1,8 @@
 import { Documentation } from "pages/Registration/types";
 
-export type FormValues = Omit<Documentation, "level"> & { level: never };
+type Key = keyof Documentation;
+const _level: Key = "level";
+
+export type FormValues = Omit<Documentation, typeof _level> & {
```
Comment (misicnenad) [inline src/pages/Registration/Steps/Documentation/types.ts:6]: Field `level` redundant. 
```ts
export type FormValues = Omit<Documentation, typeof _level> & {
 [_level]: never;
};
```
can be:
```ts
export type FormValues = Omit<Documentation, typeof _level>;
```

**Review by misicnenad — CHANGES_REQUESTED** [review]

**Comment by ap-justin** [comment]
closing - too many conflicts on destination branch

File: src/pages/Registration/Steps/Documentation/types.ts (hunk 2 above)
Comment (ap-justin) [reply src/pages/Registration/Steps/Documentation/types.ts]: fixed https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1789/commits/422182ed8381409fe2d2b8c41a7a1a5a9e44b216

File: src/pages/Registration/ConfirmEmail.tsx [hunk 3]
```diff
@@ -21,6 +21,8 @@ export default function ConfirmEmail({ classes = "" }: { classes?: string }) {
     return <Navigate to={".."} />;
   }
 
+  const openIntercom = () => (window as any).Intercom("show");
```
Comment (misicnenad) [inline src/pages/Registration/ConfirmEmail.tsx:24]:
```suggestion
  const openIntercom = () => {
    const w = (window as any);
    if ("Intercom" in w) {
      w.Intercom("show");
    }
  }
```

File: src/pages/Admin/Charity/EditProfile/schema.ts [hunk 4]
```diff
@@ -33,7 +33,7 @@ const shape: SchemaShape<FormValues> = {
   }),
   name: requiredString,
   overview: requiredString,
-  active_in_countries: Yup.array().min(1, "required"),
+  active_in_countries: Yup.array(),
```
Comment (misicnenad) [inline src/pages/Admin/Charity/EditProfile/schema.ts:36]: Is ` Yup.array()` necessary or not?

**Pull request merged**